### PR TITLE
[server] Default scale/mupmm on GetLegendGraphics 

### PR DESCRIPTION
**Pull request by signedav**
Backport from #9775

### CODE REVIEW

File: src/server/qgsserverprojectutils.h [hunk 1]
```diff
@@ -154,6 +154,14 @@ namespace QgsServerProjectUtils
    */
   SERVER_EXPORT int wmsMaxAtlasFeatures( const QgsProject &project );
 
+  /**
+   * Returns the default number of map units per millimeters in case of the scale is not given
+   * \param project the QGIS project
+   * \returns the default number of map units per millimeter
+   * \since QGIS 3.8
```
Comment (pblottiere) [inline src/server/qgsserverprojectutils.h]: Considering the backport, it should be 3.4 in this case

**Comment by pblottiere** [comment]
Hi @signedav,

Once updated (`3.8` -> `3.4`), I'll merge your PR. Do not hesitate to ping me :)

**Comment by signedav** [comment]
So I should change the comment on master as well to `\since QGIS 3.4` right?

**Comment by pblottiere** [comment]
> So I should change the comment on master as well to \since QGIS 3.4 right?

Yes. Actually, as soon as your PR is backported to 3.4, new methods have to be tagged with `\since 3.4`. Sorry I didn't catch that on your original work...

I can totally take care of updating master if you want?

**Comment by signedav** [comment]
was no effort to make a push again #9858

**Pull request merged**